ministryofjustice/laa-apply-for-criminal-legal-aid
# Set the proper aria-sort value

## Description of change
According to the documentation for `aria-sort` [1], the values should be `ascending`, `descending` or `none`.

Added a few more tests.

[1] https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-sort

## How to manually test the feature
Do an assets clobber and then `rails dartsass:build` to ensure the styles are correct.
